package.json

```diff
@@ -53,7 +53,7 @@
   },
   "peerDependencies": {
     "protobufjs": "^6.8.8",
-    "three": ">=0.96.0"
+    "three": "^0.110.0"
```

Comment:
Hm - are we requiring ThreeJS 110?

Reply:
Sorry, that was from the other commit, which should not have been part of this PR.